index.html

```diff
@@ -27,13 +33,17 @@ <h1>I'm Mustafa <br> Glad to see you!</h1>
                 <i class='fa fa-twitter'></i>
                 <i class='fa fa-linkedin'></i>
                 <i class='fa fa-medium'></i>
-                <i class='fa fa-angellist'></i>
                 <i class='fa fa-github'></i>
+                <i class='fa fa-angellist'></i>
             </ul>
         </section>
         <section class="works">
```

Comment:
- [ ] Would you mind **checking the spacing between your hero section and your cards?** It appears to be **slightly smaller than in the[ given template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122).** Adjusting this **will greatly enhance the overall look of your project.** 💪

- [ ] Please take note that **you're currently displaying all your project images to the left.** According to the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122), it is recommended **to alternate the alignment, starting with the image on the left, then right, and so forth.** Making this adjustment will ensure better consistency with the provided design. 💯

- [ ] Could you kindly _review the spacing between your project cards to ensure they align precisely with the template?_ Fine-tuning the distance between the cards **will result in a more visually pleasing layout.** 😉
   
   ### Your design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/7be8e830-558f-4e42-b055-2f95c5ddddc5)

   ### Figma design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/4de15ba0-80f1-44c4-959f-65e0bf436157)